File: app/src/main/java/com/lkms/data/model/java/SDS.java
package com.lkms.data.model.java;

import com.google.gson.annotations.SerializedName;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * Đại diện cho một hàng trong bảng "SDS".
 */
@Data
@NoArgsConstructor
@AllArgsConstructor
public class SDS {

    @SerializedName("sdsId")
    private String sdsId;

    @SerializedName("url")
    private String url;
}

File: app/src/main/java/com/lkms/data/model/SDS.kt
// Trong file: data/model/SDS.kt
package com.lkms.data.model

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

/**
 * Đại diện cho một hàng trong bảng "SDS".
 */
@Serializable
data class SDS(
    @SerialName("sdsId")
    val sdsId: String,

    @SerialName("url")
    val url: String? = null
)